Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs

```diff
@@ -152,6 +152,24 @@ genderBannerConfig state =
       }
   in config'
 
+
+-------------------------------------updateAppConfig------------------------------------
+appUpdateBannerConfig :: ST.HomeScreenState -> Banner.Config
+appUpdateBannerConfig state = 
+  let 
+    config = Banner.config
+    config' = config
```

Comment:
replace with `Banner.config  { 
        backgroundColor = Color.blue600
      , title = (getString APP_UPDATE_AVAILABLE)
      , titleColor = Color.blue800
      , actionText = (getString UPDATE_NOW_APP)
      , actionTextColor = Color.blue800
      , imageUrl = "ny_ic_app_update, https://assets.juspay.in/beckn/nammayatri/nammayatricommon/images/ny_ic_app_update.png" 
      , isBanner = state.props.isUpdateBanner
      }` instead of `let in`